app/lib/feed_manager.rb

```diff
@@ -248,7 +248,8 @@ def phrase_filtered?(status, receiver_id, context)
 
     !combined_regex.match(Formatter.instance.plaintext(status)).nil? ||
       (status.spoiler_text.present? && !combined_regex.match(status.spoiler_text).nil?) ||
-      (status.preloadable_poll && !combined_regex.match(status.preloadable_poll.options.join("\n\n")).nil?)
+      (status.preloadable_poll && !combined_regex.match(status.preloadable_poll.options.join("\n\n")).nil?) ||
+      !combined_regex.match(status.media_attachments.map(&:description).join("\n\n")).nil?
```

Comment:
Not sure why we're running the same regex 4 times instead of making a single string and running it once here

Reply:
I can't see what's failing in the test because CircleCI is requesting permission to access all of my data if I want to view the logs.